spec/features/evidence_spec.rb

```diff
@@ -76,6 +76,9 @@
       include_examples "recover deleted item", :evidence
       include_examples "recover deleted item without node", :evidence
     end
+
+    let(:model) { @evidence }
+    include_examples "nodes pages breadcrumbs", :show, Evidence
```

Comment:
```suggestion
    include_examples 'nodes pages breadcrumbs', :show, Evidence
```
Use single quotes here. Also make this adjustment for the changes below.

Reply:
Fixed!